website/siteConfig.js

```diff
@@ -2,6 +2,8 @@
 // site configuration options.
 
 const repoUrl = "https://github.com/scalacenter/scalafix";
+const metadocUrl =
+  "https://scalameta.org/metadoc/#/scalafix/scalafix-core/src/main/scala/scalafix/patch/Patch.scala#L19C23-L19C28";
```

Comment:
Is linking directly to Patch intentional?

It's a bit arbitrary if so, but I understand it's as good as any other point to get started.

It would actually be nice to link to sources directly from the specific docs sections.

Reply:
Also, is metadoc continuously updated? If not, it would be a bit strange.

Reply:
> It would actually be nice to link to sources directly from the specific docs sections.

That's a good idea! But

> Also, is metadoc continuously updated? If not, it would be a bit strange.

It's not, but I have a script that builds scalameta master + scalafix master and re-generates the site. It could be automated with a daily Travis cron job but I think the scalafix/scalameta sources will be stable enough in the coming weeks that a manual approach will do fine for now. Right now I care most about getting metadoc into the hands of users so they can try it out and report feedback.